src/common/featuremanager/FeatureManagerService.js

```diff
@@ -986,6 +986,10 @@
           if (goog.isDefAndNotNull(source.getGetFeatureInfoUrl)) {
             validRequestCount++;
           }
+          // include special case for search layer and results
+          else if (layer.get('metadata').searchLayer || layer.get('metadata').searchResults) {
+            validRequestCount++;
+          }
```

Comment:
This is a workaround to get the feature manager to display the search layer and pinned searches - any better way to do this?